recipes/types-s3transfer/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "types-s3transfer" %}
+{% set version = "0.6.0.post1" %}
+{% set lib_version = version.split(".post")[0] %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/types-s3transfer-{{ version }}.tar.gz
+  sha256: 78479647a44c9f94ca4c0d2b90f8da77013daf35485c8786b618136fb7b4501b
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+outputs:
```

Comment:
is there a reason why you are using the split package syntax? It really complicates recipes

Reply:
For some reason `test.script` is not accepted, but `ouputs.*.tests.script` works. Could please help me to rewrite it properly?

Reply:
FIxed, thank you!